rust/src/lib.rs

```diff
@@ -37,17 +35,14 @@ pub type TargetTree = rstar::RTree<GeomWithData<CachedEnvelope<TarLine>, (usize,
 /// Represents a partial source <-> target match
 #[derive(Debug, Clone)]
 pub struct MatchCandidate {
-    /// The index of the target geometry as an i32
-    pub index: i32,
+    /// The index of the source geometry
```

Comment:
These docs were wrong, hence my confusion in the other PR. I've also renamed `index` to `source_index` to be even more clear.